### PR TITLE
484.noise limits

### LINKED ISSUE #484: Noise limits leak out of the Dilation API
Currently, the 65KiB limit of Noise messages accidentally leaks out of the L2 abstractions. It should not.

### PR DESCRIPTION
**Pull request by meejah**
Fixes #484

### CODE REVIEW

**Comment by piegamesde** [comment]
I'd like to treat the protocols repository as authoritative, see also https://github.com/magic-wormhole/magic-wormhole-protocols/issues/30

**Comment by meejah** [comment]
(It would also be interesting to have Rust details, too, IMO ... is there some tool that can output different formats from unified source or similar?)

**Comment by piegamesde** [comment]
The only thing I'm aware of is rustdoc, which pretty-renders source code with documentation comments. So far I've been mostly concerned with documenting public API / library usage, but I've also seen people have no-code modules purely for documentation purposes …

File: docs/dilation-protocol.md [hunk 1]
```diff
@@ -256,6 +264,39 @@ After sending their KCM, the Follower will wait for either an empty KCM (at whic
...
+- KCM: `0x00`
+- PING: `0x01`
+- PONG: `0x02`
+- OPEN: `0x03`
+- DATA: `0x04`
+- CLOSE: `0x05`
+- ACK: `0x06`
+
+Every message starts with its tag.
+Following the is a message-specific encoding.
```
Comment (vu3rdd) [inline docs/dilation-protocol.md:282]: Following the "tag"?

**Comment by meejah** [comment]
> The only thing I'm aware of is rustdoc, [..]

I meant kind of the opposite? That is, including Rust examples in the Dilation spec / documentation (ideally you'd press some button that switches everything to Rust or Python -- or Haskell).

**Review by vu3rdd — APPROVED** [review]

**Pull request merged**